packages/pangea-node-sdk/src/types.ts

```diff
@@ -1124,14 +1203,6 @@ export namespace Vault {
       secret: string;
     }
 
-    export interface GenerateRequest
```

Comment:
Why these were removed? Aren't they used?

Reply:
They are not used.